web/templatetags/timezone_filters.py

```
@@ -0,0 +1,75 @@
+from datetime import datetime
+
+from django import template
+from django.utils import timezone
+from django.utils.timezone import is_aware, localtime, make_aware
+
+register = template.Library()
+
+
+@register.filter(name="localtime")
+def localtime_filter(value):
```

Ruff (0.8.2)

Missing return type annotation for public function `localtime_filter`

(ANN201)